test/test_autotag.py

```diff
@@ -940,13 +940,13 @@ class EnumTest(_common.TestCase):
     Test Enum Subclasses defined in beets.util.enumeration
     """
     def test_ordered_enum(self):
-        OrderedEnumTest = match.OrderedEnum('OrderedEnumTest', ['a', 'b', 'c'])
-        self.assertLess(OrderedEnumTest.a, OrderedEnumTest.b)
-        self.assertLess(OrderedEnumTest.a, OrderedEnumTest.c)
-        self.assertLess(OrderedEnumTest.b, OrderedEnumTest.c)
-        self.assertGreater(OrderedEnumTest.b, OrderedEnumTest.a)
-        self.assertGreater(OrderedEnumTest.c, OrderedEnumTest.a)
-        self.assertGreater(OrderedEnumTest.c, OrderedEnumTest.b)
+        ordered_enum = match.OrderedEnum('OrderedEnumTest', ['a', 'b', 'c'])
+        self.assertLess(ordered_enum.a, ordered_enum.b)
+        self.assertLess(ordered_enum.a, ordered_enum.c)
+        self.assertLess(ordered_enum.b, ordered_enum.c)
+        self.assertGreater(ordered_enum.b, ordered_enum.a)
+        self.assertGreater(ordered_enum.c, ordered_enum.a)
+        self.assertGreater(ordered_enum.c, ordered_enum.b)
```

Comment:
I'm not sure about this one—`OrderedEnumTest` is actually a class, not an instance. This one doesn't matter too much either way though.